lib/zendesk_apps_tools/deploy.rb

```diff
@@ -126,5 +123,11 @@ def check_job(job_id)
     rescue Faraday::Error::ClientError => e
       say_error_and_exit e.message
     end
+
+    private
+
+    def connection
```

Comment:
Might there be a less ambiguous name for this method, e.g. `cached_connection`?

Reply:
Yeah let's do that 👍